aws/resource_aws_codebuild_project.go

```diff
@@ -179,6 +179,31 @@ func resourceAwsCodeBuildProject() *schema.Resource {
 				ValidateFunc: validateAwsCodeBuildTimeout,
 			},
 			"tags": tagsSchema(),
+			"vpc_config": {
+				Type:     schema.TypeSet,
```

Comment:
I would switch this to `Type:     schema.TypeList,`. There are subtleties between the two, but the gist being that list in this context is much easier to work with when there's only 1 item.